### ISSUE TITLE
In superGAN for contest, training loss LG = Lpercep + λLRa G + ηL1,What is the value of λ and η?  

### ISSUE THREAD

**Issue by zhuwanling** [xinntao/ESRGAN#68]

**Comment by stiansel** [comment]
The values for ESRGAN appear to be in the training options file at https://github.com/xinntao/BasicSR/blob/master/codes/options/train/train_ESRGAN.yml (see near the bottom of the file)

**Comment by zhuwanling** [comment]
> The values for ESRGAN appear to be in the training options file at https://github.com/xinntao/BasicSR/blob/master/codes/options/train/train_ESRGAN.yml (see near the bottom of the file)
I have tried that. but it is not best for PI .  Did you use these  loss functions weight in your contest in PIRM?
Thanks for your reply!
